tests/publiccloud/az_l8s_nvme.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Test nvme performance on a L8s_v2 Azure instance
+# Maintainer: qa-c team <qa-c@suse.de>
+
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+# Benchmark the given device with hdparm and fail, if the given threshold [MB/sec] is not reached
+sub benchmark_device {
+    my ($device, $threshold) = @_;
+
+    my $output = script_output("hdparm -t $device | grep 'Timing buffered disk reads'");
+    # e.g. 'Timing buffered disk reads: 15528 MB in  3.00 seconds = 5174.94 MB/sec'
+    if ($output =~ "Timing buffered disk reads: (.*)=( )*(?<throughput>[0-9]+\.[0-9]*) MB/sec") {
+        my $throughput = $+{throughput};
+        record_info("$device", "Throughput: $throughput MB/sec\nThreshold: $threshold MB/sec");
+        die "Threshold for $device not reached" if ($throughput < $threshold);
+    } else {
+        die "Unexpected hdparm output";
+    }
+}
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    die "This test only works on Azure L8s instances" unless (get_required_var('PUBLIC_CLOUD_INSTANCE_TYPE') =~ 'L(8|16|32|48|64|80)s_v2');
```

Comment:
```suggestion
    die "This test only works on Azure L instances" unless (get_required_var('PUBLIC_CLOUD_INSTANCE_TYPE') =~ 'L(8|16|32|48|64|80)s_v2');
```
So, it works on L8, L16, etc... not only in L8.

Reply:
Or maybe you should limit the regex to `L8` only, cause in the test below you do
`benchmark_device("/dev/nvme0n1", 2000);`  and 2000 is specific to L8 only, for L16 the numbers are different.

Reply:
L16sv2 will have two NVME's, so in all of those cases the performance test on the first nvme is still valid.

Reply:
ah, ok, I thought it was just one disk, but with better performance. Ok, then my first comment sill applies :)

Reply:
I added a comment at the end of the test that makes this more clear to our future selves ;-)